spec/design.md

```diff
@@ -202,19 +218,22 @@ where:
 
 ### Generics
 
-Calling `defineGeneric()` defines a new generic. It has the signature:
+Calling `newGeneric()` defines a new generic. It has the signature:
+
 ```{R}
-function(name, FUN, signature) { }
+newGeneric(name, FUN, signature)
 ```
+
 The `signature` would default to all of the arguments in
-`formals(FUN)`. The body of `FUN` would resemble S3 and S4
+`formals(FUN)[1]`. The body of `FUN` would resemble S3 and S4
```

Comment:
I'd suggest we use single dispatch by default

Reply:
Makes sense. Would you please simplify this to "The `signature` would default to names(formals(FUN))[1L]"?